### PR TITLE
Bin Vision

### PR DESCRIPTION
**Pull request by LucasBA**

### CODE REVIEW

**Comment by mattlangford** [comment]
I feel like all the past commits are going to mess things up...

File: gnc/sub8_perception/nodes/bins_2d.py [hunk 1]
```diff
@@ -53,8 +54,10 @@ def __init__(self):
         self.last_image_time = None
         self.camera_model = None
         self.pose_service = rospy.Service('vision/bin/2D', VisionRequest2D, self.request_bin)
-        self.image_sub = sub8_ros_tools.Image_Subscriber('/down_camera/image_rect_color', self.image_cb)
+        self.image_sub = sub8_ros_tools.Image_Subscriber('/down/left/image_raw', self.image_cb)
```
Comment (jpanikulam) [inline gnc/sub8_perception/nodes/bins_2d.py]: You sure you want unrectified?

File: command/sub8_missions/missions/bin_dropper_3d.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            continue
+
+        response = pose_to_numpy(response.pose.pose)
+
+        yaw = tf.transformations.euler_from_quaternion(response[1])[2]
+
+        est_target_rotations.append(yaw)
+        avg_rotation = sum(est_target_rotations) / len(est_target_rotations)
+
+        yield sub.move.set_position(response[0]).yaw_left(yaw).zero_roll_and_pitch().go(speed=SPEED)
```
Comment (jpanikulam) [inline command/sub8_missions/missions/bin_dropper_3d.py:56]: Remember positions are absolute

I might be missing something

**Comment by jpanikulam** [comment]
:thumbs:

File: gnc/sub8_perception/nodes/bins_2d.py (hunk 1 above)
Comment (LucasBA) [inline gnc/sub8_perception/nodes/bins_2d.py]: Good catch that was for testing with a bag

Comment (DSsoto) [inline gnc/sub8_perception/nodes/bins_2d.py]: If you wanna test with a bag, you should run[ image_proc/rectify](http://wiki.ros.org/image_proc).

**Comment by DSsoto** [comment]
Bring the branch up to date and we'll merge.

**Pull request merged**